ansible_collections/arista/avd/requirements-dev.txt

```diff
@@ -15,3 +15,4 @@ yamllint
 treelib
 md_toc
 natsort
+deepmerge>=1.0.1
```

Comment:
Please remove `deepmerge>=1.0.1` from requirements-dev.txt as this is already covered in requirements.txt.

```suggestion
```

Reply:
we need deepmerge here since ansible-test is only reading this file (via symlink). We have a lot of other duplicates because of that. I have tried various ways of working around this, but not found a method yet.